### PR TITLE
EVA-2552 - Study export

### PR DESCRIPTION
**Pull request by tcezard**
Basic script to export EVA's study

### CODE REVIEW

File: study_export.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        self.stage = stage
+
+    @cached_property
+    def connect(self):
+        properties = get_properties_from_xml_file(self.stage, self.properties_file)
+        pg_url = properties['eva.evapro.jdbc.url'].split('jdbc:')[-1]
+        pg_user = properties['eva.evapro.user']
+        pg_pass = properties['eva.evapro.password']
+        conn = psycopg2.connect(pg_url, user=pg_user, password=pg_pass)
+        return conn
```
Comment (apriltuesday) [inline study_export.py]: If you upgrade to pyutils >= 0.3.15 (I think), you can use `get_metadata_connection_handle`

File: study_export.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    def dump_from_database(self):
+
```
Comment (apriltuesday) [inline study_export.py:53]:
```suggestion
```

**Review by apriltuesday — APPROVED** [review]

File: study_export.py (hunk 1 above)
Comment (tcezard) [reply study_export.py]: that's right. I've added it.

File: study_export.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+class StudyExport:
+    """Abstract class to provide basic function for exporting studies to the JSON format"""
+    @staticmethod
+    def get_fields(all_fields):
+        return [
+            {'name': f, 'value': all_fields[f]} for f in all_fields
+        ]
+
+    @staticmethod
+    def get_entry(fields, cross_references=None, hierarchical_field=None):
```
Comment (sundarvenkata-EBI) [inline study_export.py:21]: This method is actually "setting" a variable to the values that were passed in and returns that entry. So suggest renaming the method.

**Review by sundarvenkata-EBI — APPROVED** [review]

File: study_export.py (hunk 3 above)
Comment (tcezard) [reply study_export.py:21]: apologies I correct this in the [wrong branch](https://github.com/EBIvariation/eva-ebi-search-export/pull/3)

**Pull request merged**